core/src/test/java/com/linecorp/armeria/internal/DefaultHttpRequestTest.java

```diff
@@ -65,7 +66,7 @@ public DefaultHttpRequestTest(boolean executorSpecified) {
     @Test
     public void abortedAggregation() {
         final Thread mainThread = Thread.currentThread();
-        final DefaultHttpRequest req = new DefaultHttpRequest(HttpHeaders.of(HttpMethod.GET, "/foo"), true);
+        final HttpRequestWriter req = new DefaultHttpRequest(HttpHeaders.of(HttpMethod.GET, "/foo"));
```

Comment:
Since this is `DefaultHttpRequestTest`, it might make sense to use the concrete class here.

Reply:
Agreed. Fixed :-)